shared/rspec/support/retry.rb

```diff
@@ -6,8 +6,8 @@
   # show exception that triggers a retry if verbose_retry is set to true
   config.display_try_failure_messages = true
 
-  # run retry only on features tests
-  config.around :each, :js do |example|
+  # run retry only on systems tests
+  config.around :each, type: :system do |example|
     example.run_with_retry retry: 3
```

Comment:
How about adding an env var to do the retry only when needed e.g. CI -> `TEST_ENABLE_RETRY`. 

```
example.run_with_retry retry: 3 if ENV.fetch('TEST_ENABLE_RETRY`, true)
```

Usually in local dev, we don't want to retry as we want it to fail fast. But in the CI env, to prevent false negative we usually enable it.

I think this has be done in a few projects. What do you think?

Reply:
Agree 👍